uiTests/src/test/resources/scenarios-and-utils.rhino.js

```diff
@@ -0,0 +1,92 @@
+
+importClass(java.lang.Runnable);
```

Comment:
Yes, that's JavaScript and yes, that's Java... at the same time :exploding_head: 
Rhino's a perfect tool to troll IT recruiters :trollface:

Reply:
:xd: